config/config-observability.yaml

```diff
@@ -85,19 +85,40 @@ data:
   
     # metrics.backend-destination field specifies the system metrics destination.
     # It supports either prometheus (the default) or stackdriver.
-    # Note: Using stackdriver will incur additional charges
+    # Note: Using stackdriver will incur additional charges.
     metrics.backend-destination: prometheus
 
     # metrics.request-metrics-backend-destination specifies the request metrics
     # destination. If non-empty, it enables queue proxy to send request metrics.
     # Currently supported values: prometheus, stackdriver.
     metrics.request-metrics-backend-destination: prometheus
 
-    # metrics.stackdriver-project-id field specifies the stackdriver project ID. This
-    # field is optional. When running on GCE, application default credentials will be
+    # metrics.stackdriver-project-id field specifies the stackdriver project ID.
+    # This field is optional. When running on GCE, application default credentials will be
     # used if this field is not provided.
+    #
+    # To use a Stackdriver project ID that does not match the application default credentials,
+    # add a Secret named "stackdriver-service-account-key" to the "knative-serving" namespace.
+    # The Secret should contain the key of a Google Service Account with access scope of
+    # monitoring.metricDescriptors.create permissions to the project specified by
```

Comment:
Does create permission include read and write automatically? Also should we use singular for `permissions`?